tests/LocaleTest.php

```diff
@@ -19,8 +19,8 @@ public function testException()
 
     public function testGetPath()
     {
-        $path = implode(DIRECTORY_SEPARATOR, [dirname(__DIR__), 'src', '..', 'locale']).DIRECTORY_SEPARATOR;
-        $this->assertEquals($path, Locale::getPath());
+        $rootDir = realpath(dirname(__DIR__).'/src/..');
```

Comment:
Test may fail having `/` here

Reply:
Provide a test enviroment. I do not like `DIRECTORY_SEPARATOR` as it is very long and...

can you even imagine using it like: require `__DIR__ . DIRECTORY_SEPARATOR . 'example.php'`. I do not.